### ISSUE TITLE
[Bug]Crash after tapping 🛈 icon from uBlock settings

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#8142]
### Prerequisites
Have uBlock Origin installed.

## Steps to reproduce
1. From the Add-ons manager tap on uBlock Origins
2. Tap and open "Settings"
3. From the Settings section tap one of the 🛈 icons

### Expected behavior
The link should be properly opened.

### Actual behavior
Native code crash.

### Device information

* Android device: 
• Google Pixel 3a XL (Android 10)
• Google Pixel 3a (Android 9)
• Huawei Mate 20 Lite (Android 8.1.0)
• Samsung Galaxy S7 (Android 7.0.0)
• OnePlus A3 (Android 6.0.1)
• LG Nexus 4 (Android 5.1.1)

* Fenix version: Nightly #20370604 from 2/6

### Notes
<details>
    <summary>Video comparison between Fennec VS Fenix (dropdown)</summary>

► Fenix
![20200206-095950](https://user-images.githubusercontent.com/51314259/73918006-52066a80-48c9-11ea-9075-ad1a4afd82aa.gif)

►Fennec
![20200206-100506](https://user-images.githubusercontent.com/51314259/73917977-3b601380-48c9-11ea-96cf-f2e564c37978.gif)

</details>

►Log
```kotlin
02-06 09:59:58.096 16881-16894/? E/mozac/CrashReporter: CrashHandlerService received native code crash
02-06 09:59:58.096 16881-16894/? I/mozac/CrashReporter: Received crash: NativeCodeCrash(minidumpPath=/data/user/0/org.mozilla.fenix.nightly/files/mozilla/b41h6vuu.default/minidumps/5d4299aa-eac1-9626-ebd2-a543ec91e223.dmp, minidumpSuccess=false, extrasPath=/data/user/0/org.mozilla.fenix.nightly/files/mozilla/b41h6vuu.default/minidumps/5d4299aa-eac1-9626-ebd2-a543ec91e223.extra, isFatal=true, breadcrumbs=[])
02-06 09:59:58.097 16881-16894/? I/mozac/CrashReporter: Showing prompt
02-06 09:59:58.099 1279-2222/? I/ActivityManager: START u0 {flg=0x10018000 cmp=org.mozilla.fenix.nightly/mozilla.components.lib.crash.prompt.CrashReporterActivity (has extras)} from uid 10210 pid 16881 on display 0
02-06 09:59:58.131 15700-15700/? D/AppTracker: App Event: stop
02-06 09:59:58.137 16881-16894/? I/art: System.exit called, status: 0
02-06 09:59:58.137 16881-16894/? I/AndroidRuntime: VM exiting with result code 0, cleanup skipped.
02-06 09:59:58.170 1279-1835/? V/BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@c89d55d
02-06 09:59:58.170 1279-1835/? I/ActivityManager: Start proc 16896:org.mozilla.fenix.nightly:mozilla.components.lib.crash.CrashReporter/u0a210 for activity org.mozilla.fenix.nightly/mozilla.components.lib.crash.prompt.CrashReporterActivity
02-06 09:59:58.172 481-481/? E/lowmemorykiller: Error writing /proc/15700/oom_score_adj; errno=22
02-06 09:59:58.173 1279-2089/? I/ActivityManager: Process org.mozilla.fenix.nightly:mozilla.components.lib.crash.CrashHandler (pid 16881) has died
02-06 09:59:58.174 1279-2089/? W/ActivityManager: Scheduling restart of crashed service org.mozilla.fenix.nightly/mozilla.components.lib.crash.handler.CrashHandlerService in 1000ms
02-06 09:59:58.194 1279-1328/? V/ActivityManager: killProcessGroupAsync took 20 ms for PID 16881 on thread 14
02-06 09:59:58.272 1279-1818/? D/GraphicsStats: Buffer count: 9
02-06 09:59:58.272 1279-2803/? I/WindowState: WIN DEATH: Window{8a05ed2 u0 SurfaceView}
02-06 09:59:58.272 1279-2802/? I/ActivityManager: Process org.mozilla.fenix.nightly (pid 15700) has died
02-06 09:59:58.275 15770-15770/? I/ServiceChildProcess: Service has been unbound. Stopping.
02-06 09:59:58.276 15787-15787/? I/ServiceChildProcess: Service has been unbound. Stopping.
02-06 09:59:58.277 1279-2222/? I/WindowState: WIN DEATH: Window{e8e4e19 u0 org.mozilla.fenix.nightly/org.mozilla.fenix.nightly.App}
02-06 09:59:58.285 15770-15770/? I/Process: Sending signal. PID: 15770 SIG: 9
02-06 09:59:58.332 1279-1328/? V/ActivityManager: killProcessGroupAsync took 56 ms for PID 15700 on thread 14
02-06 09:59:58.333 724-724/? I/Zygote: Process 15700 exited cleanly (11)
02-06 09:59:58.335 1279-2089/? I/ActivityManager: Process org.mozilla.fenix.nightly:socket (pid 15787) has died
02-06 09:59:58.336 1279-1328/? V/ActivityManager: killProcessGroupAsync took 1 ms for PID 15787 on thread 14
02-06 09:59:58.337 1279-2578/? I/ActivityManager: Process org.mozilla.fenix.nightly:tab (pid 15770) has died
02-06 09:59:58.343 1279-1328/? V/ActivityManager: killProcessGroupAsync took 7 ms for PID 15770 on thread 14
02-06 09:59:58.415 16896-16896/? D/AppTracker: App Event: start
02-06 09:59:58.430 16896-16912/? D/OpenGLRenderer: Use EGL_SWAP_BEHAVIOR_PRESERVED: true
02-06 09:59:58.467 16896-16912/? I/Adreno: QUALCOMM build                   : 19f8f74, Iac38d51750
    Build Date                       : 06/06/16
    OpenGL ES Shader Compiler Version: XE031.07.00.01
    Local Branch                     : case2489916_Unmap_crash
    Remote Branch                    : 
    Remote Branch                    : 
    Reconstruct Branch               : 
02-06 09:59:58.471 16896-16912/? I/OpenGLRenderer: Initialized EGL, version 1.4
02-06 09:59:58.485 16896-16896/? D/AbstractTracker: Event success
02-06 09:59:58.497 16896-16912/? E/HAL: load: gralloc.msm8996.so
02-06 09:59:58.536 1279-1352/? I/ActivityManager: Displayed org.mozilla.fenix.nightly/mozilla.components.lib.crash.prompt.CrashReporterActivity: +395ms
```

❓ Same as #8140

**Comment by csadilek** [comment]
> ❓ Same as #8140

@AndiAJ yes, this and #8140 have the same cause as #8117. Do you want to track separate or should we just keep #8117 which is waiting for a GV fix: https://bugzilla.mozilla.org/show_bug.cgi?id=1613237

**Comment by AndiAJ** [comment]
@csadilek - I would opt to keep them separate, if you agree. ☺️ 
Just want to be sure we don't overlook any of them while testing.

**Comment by concastinator** [comment]
The browser also crashes if one presses the "(i)" button (presumably standing for "information") next to the "AdGuard annoyances" and "AdGuard social media" in the "Filters list" tab of the uBlock settings. Since I believe the cause to be the same as for the bugs mentioned in the posts above, I'm writing this here.

**Comment by csadilek** [comment]
@AndiAJ This crash should be fixed in the latest Nightly. See also: https://github.com/mozilla-mobile/fenix/issues/8140#issuecomment-589961056

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **`Nightly Build #20550606 from 02/24`** using the following devices:
• Google Pixel 3a XL (Android 10)
• Google Pixel 3a (Android 9)
• Huawei Mate 20 Lite (Android 8.1.0)
• Samsung Galaxy S7 (Android 7)
• OnePlus A3 (Android 6.0.1)
• LG Nexus 4 (Android 5.1.1)

► Video
![20200224-113319](https://user-images.githubusercontent.com/51314259/75141549-d9414400-56f9-11ea-99e2-fe3a20c5d25b.gif)
